### PR TITLE
feat: added top 3 rankings for leaderboard

### PR DESCRIPTION
**Pull request by keithcharlesc**
- Top 3 Rankings Component
- Preview at http://localhost:3000/leaderboard (Yet to be linked to dashboard)

### CODE REVIEW

File: pages/leaderboard/index.tsx [hunk 1]
```diff
@@ -20,14 +21,17 @@ const LeaderboardPage = () => {
...
+        </div>
+
         <div>
-          <br></br>
-          <br></br>
           <br></br>
           <div className='flex justify-between'>
             <strong>Rankings this month</strong>
-            <button className='text-sm font-semibold mr-2 text-gray-500'>View More {'>'}</button>
+            <button className='text-sm font-semibold mr-2 text-gray-500'>View More {'＞'}</button>
```
Comment (remuskwan) [inline pages/leaderboard/index.tsx:34]: Use ChevronRight icon from `@heroicons/outline`

File: pages/leaderboard/index.tsx [hunk 2]
```diff
@@ -20,14 +21,17 @@ const LeaderboardPage = () => {
...
+        <h2 className='text-md font-bold leading-6 text-gray-900 mb-6'>Leaderboard</h2>
+
+        <div className='flex justify-evenly'>
+          <LeaderboardTopRankings />
+        </div>
+
         <div>
-          <br></br>
-          <br></br>
           <br></br>
```
Comment (remuskwan) [inline pages/leaderboard/index.tsx:31]: I think can remove this too haha

**Review by remuskwan — CHANGES_REQUESTED** [review]
Looks good except for a few nits!

File: pages/leaderboard/index.tsx (hunk 2 above)
Comment (keithcharlesc) [reply pages/leaderboard/index.tsx]: @remuskwan Pushed a new update for the requested changes!

**Review by remuskwan — APPROVED** [review]
LGTM!

**Pull request merged**